bindings/pyroot_experimental/PyROOT/python/ROOT/pythonization/_rdataframe.py

```diff
@@ -74,16 +89,80 @@ def RDataFrameAsNumpy(df, columns=None, exclude=None):
     return py_arrays
 
 
+# Pythonizations for the following methods:
```

Comment:
Following the style of other pythonizations, all the documentation of the pythonization should be inside the function itself, with this format:
https://github.com/root-project/root/blob/master/bindings/pyroot_experimental/PyROOT/python/ROOT/pythonization/_th1.py#L16

Before the `Parameters` section you can add a paragraph explaining what this function does, which is: `Check whether the user called one of the HistoXD or ProfileXD methods of RDataFrame with a tuple as first argument. In that case, extract the tuple items to construct a model object and call the original implementation of the method with that object`. That is all you need to say.